### PR TITLE
chore(sitemap): sitemaps refresh

### PR DESCRIPTION
**Pull request by kptdobe**
Refreshed all the sitemaps.

### CODE REVIEW

**Review by rofe — APPROVED** [review]

File: sitemap.br.xml [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+  <url>
+    <loc>https://blog.adobe.com/br/publish/2021/04/08/por-que-ser-um-cidadao-global-aumenta-a-criatividade.html</loc>
+    <lastmod>2021-04-08</lastmod>
+  </url>
+  <url>
+    <loc>https://blog.adobe.com/br/publish/2021/03/28/como-ter-um-curriculo-vitae-criativo-e-impressionante-modelos-gratuitos.html</loc>
+    <lastmod>2021-03-28</lastmod>
+  </url>
+  <url>
+    <loc>https://blog.adobe.com/br/publish/2021/02/25/como-ter-um-curriculo-vitae-criativo-e-impressionante-modelos-gratuitos.html</loc>
```
Comment (rofe) [inline sitemap.br.xml:12]: Isn't this a duplicate of the entry above?

Comment (rofe) [reply sitemap.br.xml:12]: Actually this one doesn't exist anymore... maybe a move that got stuck in the index? @dominique-pfister

Comment (kptdobe) [reply sitemap.br.xml]: The eye of the tiger ;)
The BR index was outdated, I re-published it and it fixed this duplicate.

Comment (rofe) [reply sitemap.br.xml]: Happened to be the first file in the review 😄

**Pull request merged**